classes/Hook.php

```diff
@@ -1260,13 +1267,19 @@ private static function getMethodName(string $hookName): string
      *
      * @return bool
      */
-    public static function getHookStatusByName($hook_name): bool
+    public static function getHookStatusByName($hook_name, $refreshCache = false): bool
```

Comment:
We could avoid `BC Break` if we do:

`Hook::resetStaticCache()`
`Hook::getHookStatusByName('someName');`

Reply:
Ok, now I understand why you decided to go with the `clearCache` instead of the `resetStaticCache`, although I'm still not sure if we really need a new argument here, if we target `1.7.8.x` it might be better to keep it simple

Reply:
@kpodemski What would you like me to do now? And why is this a `BC break`?
- remove parameter `$refreshCache` ? I decided to do it like this, because it was done like this in another class.
- extend `ObjectModel::resetStaticCache` in `Hook::resetStaticCache` ?

The tests fail at `LegacyTests\Unit\Adapter\Module\PrestaTrust\PrestaTrustCheckerTest::testModuleHasPico`, this really looks unrelated.

Reply:
@lmeyer1 

If someone has an override for this method, it won't work anymore after the upgrade (strict error).

I would keep `Hook::clearCache` but remove `$refreshCache`. In the tests, we can rely on the `Hook::clearCache`

Reply:
@kpodemski Thanks for clarification.